tests/scripts/test_psa_compliance.py

```diff
@@ -50,12 +50,9 @@ def main(library_build_dir: str):
 
     in_tf_psa_crypto_repo = build_tree.looks_like_tf_psa_crypto_root(root_dir)
 
-    if in_tf_psa_crypto_repo:
-        crypto_name = 'tfpsacrypto'
-        library_subdir = 'core'
-    else:
-        crypto_name = 'mbedcrypto'
-        library_subdir = 'library'
+    crypto_name = build_tree.project_crypto_name()
+
+    library_subdir = build_tree.crypto_core_directory()
```

Comment:
Let's be consistent:
```suggestion
    crypto_name = build_tree.project_crypto_name(root_dir)
    library_subdir = build_tree.crypto_core_directory(root_dir)
```
(Although this script doesn't work unless called from the mbedtls/tfpsacrypto root anyway, so this only changes the error that you get if you run it from a subdirectory. No need to fix that now.)